### PR TITLE
Add new OAuth2 settings to credentials-related playbooks.

### PR DESCRIPTION
**Pull request by doctoryes**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: playbooks/roles/oauth_client_setup/defaults/main.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ oauth_client_setup_oauth2_clients:
     - {
         name: "{{ credentials_service_name | default('None') }}",
         url_root: "{{ CREDENTIALS_URL_ROOT | default('None') }}",
-        id: "{{ CREDENTIALS_SOCIAL_AUTH_EDX_OIDC_KEY | default('None') }}",
-        secret: "{{ CREDENTIALS_SOCIAL_AUTH_EDX_OIDC_SECRET | default('None') }}",
+        id: "{{ CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_KEY | default('None') }}",
+        secret: "{{ CREDENTIALS_SOCIAL_AUTH_EDX_OAUTH2_SECRET | default('None') }}",
```
Comment (doctoryes) [inline playbooks/roles/oauth_client_setup/defaults/main.yml]: @nasthagiri @pwnage101 Should this be the new client credentials OAuth2 key/secret instead?

**Comment by doctoryes** [comment]
@nasthagiri @pwnage101 I added the new client credentials key/secret everywhere I think it should go here...

File: playbooks/roles/oauth_client_setup/defaults/main.yml (hunk 1 above)
Comment (pwnage101) [reply playbooks/roles/oauth_client_setup/defaults/main.yml]: Under the hood, the ansible task for this role uses the create_oauth2_client management command which creates a DOP client, not a DOT app which this change would actually want.

I'm not yet totally sure what the solution is here, but it seems like we should modify the role to add a new ansible task for creating DOT apps using the create_dot_application management command.

Comment (doctoryes) [reply playbooks/roles/oauth_client_setup/defaults/main.yml]: That seems reasonable - I'll look into it.

**Review by nasthagiri — APPROVED** [review]

**Review by pwnage101 — APPROVED** [review]

**Pull request merged**